File: VB/DXSample/CustomTableView.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.Xpf.Grid

Namespace DXSample
	Public Class CustomTableView
		Inherits TableView
		Public Sub New()
			MyBase.New()

		End Sub
		Protected Overrides Function CreateEmptyDropTarget() As DevExpress.Xpf.Core.IDropTarget
			Return New CustomRemoveColumnDropTarget()
		End Function
	End Class
End Namespace

File: CS/DXSample/CustomTableView.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.Xpf.Grid;

namespace DXSample
{
    public class CustomTableView : TableView
    {
        public CustomTableView(): base()
        {

        }
        protected override DevExpress.Xpf.Core.IDropTarget CreateEmptyDropTarget()
        {
            return new CustomRemoveColumnDropTarget();
        }
    }
}
